<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Contours"/>
    <addaction name="separator"/>
    <addaction name="actionMake_traindata"/>
   </widget>
   <widget class="QMenu" name="menuVerarbeiten">
    <property name="title">
     <string>Verarbeiten</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStart_Neural_Network_Training"/>
    <addaction name="actionStop_Neural_Network_Training"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionNeural_Network_Settings"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuSettings"/>
   <addaction name="menuVerarbeiten"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="log_listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start..</string>
   </property>
   <property name="shortcut">
    <string comment="F5">F5</string>
   </property>
  </action>
  <action name="actionSave_Contours">
   <property name="text">
    <string>Save Contours</string>
   </property>
  </action>
  <action name="actionNeural_Network_Settings">
   <property name="text">
    <string>Neural Network Settings</string>
   </property>
  </action>
  <action name="actionStart_Neural_Network_Training">
   <property name="text">
    <string>Start Neural Network Training</string>
   </property>
  </action>
  <action name="actionStop_Neural_Network_Training">
   <property name="text">
    <string>Stop Neural Network Training</string>
   </property>
  </action>
  <action name="actionMake_traindata">
   <property name="text">
    <string>Make traindata</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
